openmc/deplete/abc.py

```diff
@@ -638,9 +634,8 @@ def __init__(
                     "this uses {}".format(
                         self.__class__.__name__, res.data.shape[0],
                         self._num_stages))
-        else:
-            # if the user specifies a continue run without a previous results, set the flag to False for them
-            continue_timesteps = False
+        elif continue_timesteps:
+            raise ValueError("Continuation run requires passing prev_results.")
```

Comment:
I could see a reason for preferring this behavior from a logic perspective, but I was hoping we could do it like this
```
else:
    continue_timesteps = False
```
so that  `continue_timesteps = True` for a case with no previous results would be allowed. That way you can truly use the same script as the initial run. Just some food for thought.

Reply:
I understand where you're coming from, but I do prefer to have it be logically consistent: if the user says timesteps should be continued, there needs to be something to continue from.

Reply:
Fair enough! I can live with two slightly different scripts 😄